_health-care/_js/_components/address-state.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import _ from 'lodash';
+
+class AddressState extends React.Component {
+  constructor() {
+    super();
+    this.state = { hasError: false };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const state = this.refs.StateInput.value;
```

Comment:
Yes, this is a bug. Not sure why it didn't fail, but I just noticed that using `this.refs.StateInput.value` kept the form from persisting the field value. Will correct in next push.